src/dvc_objects/_tqdm.py

```diff
@@ -100,6 +100,10 @@ def __init__(
             total=total,
             **kwargs,
         )
+        
+        if not hasattr(self, "desc"):
+            self.desc = desc or ""
+            
```

Comment:
```suggestion
        # NOTE: temporary workaround, see
        # https://github.com/tqdm/tqdm/pull/1341
        if not hasattr(self, "desc"):
            self.desc = desc or ""

```

But lets wait for https://github.com/tqdm/tqdm/pull/1341 , if it goes through, the release will follow promptly, so we might just need to upgrade tqdm requirements instead and that would be it 🙂

Reply:
As per https://github.com/iterative/dvc/issues/7896#issuecomment-1179182462 not sure this PR is actually a solution. Different repo (https://github.com/iterative/dvc-data/pull/92) instead?

Reply:
> Thanks @sjawhar! However as per [iterative/dvc#7896 (comment)](https://github.com/iterative/dvc/issues/7896#issuecomment-1179182462) not sure this PR is actually a solution. Different repo ([iterative/dvc-data#92](https://github.com/iterative/dvc-data/pull/92)) instead?

Both are solutions (dvc-data still uses this dvc-objects class), but I prefer yours :+1:

Reply:
Ah https://github.com/iterative/dvc-data/blob/d8c407aaaa9b1f550214610ccb609cd373358ad1/src/dvc_data/_progress.py#L1 I see indeed